handler/perm_handlers_test.go

```diff
@@ -0,0 +1,247 @@
+package handler
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"net/http/httptest"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/model"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/pkg/logging"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/service"
+	mock_service "stlab.itechart-group.com/go/food_delivery/authorization_service/service/mocks"
+	"testing"
+)
+
+func TestHandler_getPermsByRoleId(t *testing.T) {
+	type mockBehavior func(s *mock_service.MockRolePerm, id int)
+
+	testTable := []struct {
+		name                string
+		input               string
+		id                  int
+		mockBehavior        mockBehavior
+		expectedStatusCode  int    //expected code
```

Comment:
why do you need the comment? I think the meaning of this field is clear from its name